src/ssl.c

```diff
@@ -43550,7 +43752,7 @@ int wolfSSL_PEM_write_bio_PKCS7(WOLFSSL_BIO* bio, PKCS7* p7)
 }
 #endif /* defined(OPENSSL_ALL) && defined(HAVE_PKCS7) */
 
-#ifdef OPENSSL_ALL
+#if defined(OPENSSL_ALL) || defined(OPENSSL_EXTRA)
```

Comment:
Please just use `OPENSSL_EXTRA` here. When `OPENSSL_ALL` is defined `OPENSSL_EXTRA` will always be defined.